recipes/molmod/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "molmod" %}
+{% set version = "1.4.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ca0e461fe42fc8b7e0b481cda33aac247d86b5fb817b5076e20abc90dd577797
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [not linux or py<36]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - setuptools
```

Comment:
setuptools, if really needed should also be in the host section

Reply:
You can remove it from the builds section now.
And maybe you can test it completely without setuptools.

Reply:
Please remove setuptools from this section